packages/algorand-js/test/src/interpreter/opcode-list.ts

```diff
@@ -412,4 +415,445 @@ describe("Teal Opcodes", function () {
       );
     });
   });
+
+  describe("err opcode", function () {
+    const stack = new Stack<StackElem>();
+
+    it("should throw error", function () {
+      const op = new Err();
+      expectTealError(
+        () => op.execute(stack),
+        ERRORS.TEAL.TEAL_ENCOUNTERED_ERR
+      );
+    });
+  });
+
+  describe("Sha256 opcode", function () {
+    const stack = new Stack<StackElem>();
+
+    it("should return correct hash", () => {
+      stack.push(toBytes("MESSAGE"));
+      const op = new Sha256();
+      op.execute(stack);
+
+      const expected = Buffer.from(
+        "b194d92018d6074234280c5f5b88649c8db14ef4f2c3746d8a23896a0f6f3b66", 'hex');
+      const top = stack.pop() as Uint8Array;
+
+      assert.deepEqual(expected, top);
+    });
+  });
+
+  describe("Sha512_256 opcode", function () {
+    const stack = new Stack<StackElem>();
+
+    it("should return correct hash", function () {
+      stack.push(toBytes("MESSAGE"));
+      const op = new Sha512_256();
+      op.execute(stack);
+
+      const expected = Buffer.from(
+        "f876dfdffd93791dc919586232116786362d434fe59d06097000fcf42bac228b", 'hex');
+      const top = stack.pop() as Uint8Array;
```

Comment:
I don't think we need to use `as Uint8Array` (as this is supposed to be a test). Could you confirm?